agora/Agora/Proposal/Scripts.hs

```diff
@@ -443,14 +448,22 @@ proposalValidator =
             let expectedDatum =
                   mkRecordConstr
                     PProposalDatum
-                    ( #proposalId .= proposalInputDatumF.proposalId
-                        .& #effects .= proposalInputDatumF.effects
-                        .& #status .= proposalInputDatumF.status
-                        .& #cosigners .= pdata updatedSigs
-                        .& #thresholds .= proposalInputDatumF.thresholds
-                        .& #votes .= proposalInputDatumF.votes
-                        .& #timingConfig .= proposalInputDatumF.timingConfig
-                        .& #startingTime .= proposalInputDatumF.startingTime
+                    ( #proposalId
+                        .= proposalInputDatumF.proposalId
+                        .& #effects
+                        .= proposalInputDatumF.effects
+                        .& #status
+                        .= proposalInputDatumF.status
+                        .& #cosigners
+                        .= pdata updatedSigs
+                        .& #thresholds
```

Comment:
This is not looking good either.

Reply:
Yeah, it's a bit of a shame, but I don't think we can really get around it. It's a change in 0.8.2.0 fourmolu.